src/pages/send-tokens/components/send-form-inner.tsx

```diff
@@ -26,7 +26,7 @@ import {
   useSerializedTransactionPayloadState,
 } from '@store/transactions/transaction.hooks';
 import { useFeeEstimationsQuery } from '@query/fees/fees.hooks';
-import { ShowEditNonceAction } from '@pages/sign-transaction/components/show-edit-nonce';
+import { ShowEditNonceAction } from '@components/show-edit-nonce';
```

Comment:
I think I moved this too in my PR to add the high fee confirmation but I moved it in with the feature `edit-nonce-drawer`? I'm not sure it would ever be used outside of that?

Reply:
No change is needed here. All good to merge.